core/parser.mly

```diff
@@ -1130,7 +1136,7 @@ type_arg_list:
 type_arg:
 | datatype                                                     { Datatype.Type $1     }
 | braced_fieldspec                                             { Datatype.Presence $1 }
-| LBRACE row RBRACE                                            { Datatype.Row $2      }
+| LBRACE trow RBRACE                                           { Datatype.Row $2      }
```

Comment:
I think this should just be `row`.

Reply:
I just merged `trow` and `erow`.
Initially, I did this because `efields` are used in arrows so it seemed to me the best thing to use. Do you really think we should add the effect application to the `row` ? And what about the dot thing ? I think that, when it is activated, we should be able to use the dot in effect aliases, isn't it ?